html/mycakeapp/src/Controller/MainController.php

```diff
@@ -69,4 +70,36 @@ public function ajax()
       ));
     }
   }
+
+  public function top()
+  {
+    $this->viewBuilder()->setLayout('no-frame');
+    $today = Time::now();
+    //条件に当てはまるスライドショーを抽出(条件は基本設計書参照)
+    $slideshowPictures = $this->SlideshowPictures->find()
+      ->contain(['Movies'])
+      ->where(['SlideshowPictures.started_at <=' => $today])
+      ->andWhere(['SlideshowPictures.finished_at >=' => $today])
+      ->andWhere(['SlideshowPictures.is_deleted' => 0])
+      ->andWhere(['Movies.is_deleted' => 0])
+      ->order(['Movies.started_at' => 'desc'])
+      ->limit(3)
+      ->toArray();
```

Comment:
-修正不要-
見やすさの観点で指摘入れています。
動作には問題ないし、コード量もさほど変わらないので修正不要です。
参考)[CakePHP 3.9 Red Velvet Cookbook](https://book.cakephp.org/3/ja/orm/query-builder.html#select)、[CakePHP3でのクエリビルダ記法まとめ](https://www.ritolab.com/entry/67#where_base)

```suggestion
    $slideshowPictures = $this->SlideshowPictures->find()
      ->contain(['Movies'])
      ->where([
        'SlideshowPictures.started_at <=' => $today,
        'SlideshowPictures.finished_at >=' => $today,
        'SlideshowPictures.is_deleted' => 0,
        'Movies.is_deleted' => 0,
      ])
      ->order(['Movies.started_at' => 'desc'])
      ->limit(3)
      ->toArray();
```

Reply:
ありがとうございます！！
ここも勉強のために修正させていただきます！笑